### DISCUSSION TITLE
Contributing to Symfony

### DISCUSSION THREAD

**Discussion by gnito-org** [symfony/symfony#43011] [Q&A] [answered]
I'm following the documentation for [Proposing a Change](https://symfony.com/doc/current/contributing/code/pull_requests.html#check-that-the-current-tests-pass) and it says one must check on Travis-CI - https://travis-ci.org/symfony/symfony - for test failures.

That link says one must go to https://travis-ci.com/symfony/symfony, which results in a No Results Found page.

What is the current and correct procedure for that step?

**Comment by mbabker** [comment, marked as answer]
It's all using GitHub Actions now - https://github.com/symfony/symfony/actions

**Comment by javiereguiluz** [comment]
@gnito-org if you have some time, could you please submit a pull request to https://github.com/symfony/symfony-docs to fix this error? If you do, please select the `4.4` branch and we'll merge it in all supported branches. Thanks!

**Comment by gnito-org** [comment]
@javiereguiluz https://github.com/symfony/symfony-docs/pull/15818
